sdks/python/apache_beam/dataframe/frames.py

```diff
@@ -957,6 +957,69 @@ def truncate(df):
             requires_partition_by=partitionings.Arbitrary(),
             preserves_partition_by=partitionings.Arbitrary()))
 
+  @frame_base.with_docs_from(pd.DataFrame)
+  @frame_base.args_to_kwargs(pd.DataFrame)
+  @frame_base.populate_defaults(pd.DataFrame)
+  def unstack(self, **kwargs):
+    level = kwargs.get('level', -1)
+
+    if self._expr.proxy().index.nlevels == 1:
+      if PD_VERSION < (1, 2):
+        raise frame_base.WontImplementError(
+          "pandas==1.1.5 has an indexing error bug," \
+          " causing an error with unstack()")
```

Comment:
nit: I'm not sure it's helpful to reference the bug, could we change it to something like this?

```suggestion
          "unstack() is not supported when using pandas < 1.2.0" 
          "Please upgrade to pandas 1.2.0 or higher to use this operation.")
```